src/styles/main.scss

```diff
@@ -708,3 +681,73 @@ Fixing styling for the code example tables in the "Migrating From..." pages
   top: calc(50% - 11px);
   left: 18px;
 }
+
+// CSS styling for the date modifiex X ago
+.quick-info-list {
+  background: #FFFFFF;
+  border: 1px solid #DDDDDD;
+  box-sizing: border-box;
+  border-radius: 4px;
+  margin-bottom: 24px;
+  display: flex;
+}
+
+#time-posted-on {
+  padding: 10px 10px 10px 10px;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  color: #343434;
+}
+
+.quick-info-list img {
+  margin: 0px 3px 3px 0px;
+}
```

Comment:
Since this is scss you can nest the `img` selector underneath the original `.quick-info-list` declaration on line 686.